Src/ILGPU/Runtime/Cuda/CudaNativeMethods.cs

```diff
@@ -12,11 +12,22 @@
 #pragma warning disable IDE1006 // Naming Styles
 #pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
 
+using System;
+
 namespace ILGPU.Runtime.Cuda
 {
     #region Enums
 
-    enum StreamFlags
+    /// <summary>
+    /// <para>Used to create <see cref="CudaStream"/> objects.</para>
+    /// <para><see cref="CU_STREAM_DEFAULT"/> specifies a blocking stream</para>
+    /// <para>
+    /// <see cref="CU_STREAM_NON_BLOCKING"/> specifies a non-blocking stream.
+    /// It is used by default
+    /// </para>
+    /// </summary>
+    [Flags]
+    public enum StreamFlags
     {
         CU_STREAM_DEFAULT = 0,
```

Comment:
We should add some comments to the enumeration members as they have become publicly visible.
Maybe we can just convert the comments inside the `<para>...` regions and apply them to the individual enumeration members.